fastq_filtrator.py

```diff
@@ -0,0 +1,113 @@
+def read_reads(input_fastq: str) -> list:
+    with open(input_fastq) as fastq_file:
+        reads_fast = fastq_file.readlines()
+
+        all_reads = []
+        seq = 0
+        for read in reads_fast:
+            seq += 1
+            if seq % 2 == 0 and seq % 4 != 0:
+                all_reads.append(read)
+        fastq_file.close()
+
+        return all_reads
+
+
+def gc_read(reads: list, borders=(0, 100)) -> list:

```

Comment:
Название немного не соответствует действительности. Мы же считаем GC для **всех** ридов
```suggestion
def gc_reads(reads: list, borders=(0, 100)) -> list:

```